test/jdk/java/nio/channels/DatagramChannel/SendReceiveMaxSize.java

```diff
@@ -65,6 +66,7 @@
 public class SendReceiveMaxSize {
     private final static int IPV4_SNDBUF = 65507;
     private final static int IPV6_SNDBUF = 65527;
+    private final static int IPV6_SNDBUF_AIX = 65487;
```

Comment:
I really dislike this. I suspect the right thing will be to add a method to jdk.test.lib.net.IPSupport or some of the other test infra classes to avoid putting these constants in specific tests.